### PR TITLE
ci(workflows): add basic build workflow

### PR DESCRIPTION
**Pull request by steebchen**

### CODE REVIEW

File: simple-examples/src/manual-trigger.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ const hatchet = Hatchet.init();
 async function main() {
   const workflowRunId = await hatchet.admin.run_workflow("example", {});
 
+  // @ts-ignore // TODO fix this
   hatchet.listener.on(workflowRunId, async (event: StepRunEvent) => {
```
Comment (steebchen) [inline simple-examples/src/manual-trigger.ts:10]: this doesn't compile, @grutt any idea what this is supposed to look like?

Comment (steebchen) [reply simple-examples/src/manual-trigger.ts:10]: I should have just let AI do the job 😁 
image

Comment (grutt) [reply simple-examples/src/manual-trigger.ts]: 🤖

File: simple-examples/package.json [hunk 2]
```diff
@@ -14,8 +14,7 @@
     "url": "https://github.com/hatchet-dev/hatchet-typescrpt-quickstart.git"
   },
   "scripts": {
-    "tsc:build": "tsc && resolve-tspaths",
-    "generate": "./generate-protoc.sh",
```
Comment (grutt) [inline simple-examples/package.json:18]: how are we handling this generation now?

**Review by grutt — APPROVED** [review]

File: simple-examples/package.json (hunk 2 above)
Comment (steebchen) [reply simple-examples/package.json:18]: this seems just like a leftover from a copy paste, I don't think we need to generate anything here do we?

Comment (grutt) [reply simple-examples/package.json:18]: you're correct, I didn't realize we were in the quickstart -- brain was in the wrong context.

**Pull request merged**